llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1484,6 +1484,8 @@ void VPWidenCastRecipe::execute(VPTransformState &State) {
   Value *Cast = Builder.CreateCast(Instruction::CastOps(Opcode), A, DestTy);
```

Comment:
Would be nice to have a way to specify the flags at construction rather than having to add them after the fact, but it looks like the builder interface hasn't been updated yet to take the flags at construction :(